icons/nextra.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="M22.604 20.422c-2.25-2.25-3.488-5.24-3.488-8.422s1.239-6.172 3.488-8.422l.945-.944a1.545 1.545 0 0 0 0-2.182 1.545 1.545 0 0 0-2.183 0l-.944.944c-4.644 4.644-12.2 4.644-16.843 0L2.635.452A1.533 1.533 0 0 0 1.543 0c-.412 0-.8.16-1.09.452A1.533 1.533 0 0 0 0 1.543c0 .412.16.8.452 1.091l.944.944c2.25 2.25 3.489 5.24 3.489 8.422s-1.24 6.172-3.489 8.422l-.944.944A1.534 1.534 0 0 0 0 22.456c0 .413.16.8.452 1.092.292.292.68.452 1.091.452.413 0 .8-.16 1.092-.452l.944-.944c4.644-4.643 12.2-4.643 16.843 0l.944.944c.602.602 1.58.601 2.183 0a1.545 1.545 0 0 0 0-2.182l-.945-.944z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Nextra</title><path d="M22.68 21.031c-4.98-4.98-4.98-13.083 0-18.063l.978-.978c.22-.22.342-.513.342-.825 0-.311-.122-.604-.342-.824-.44-.441-1.207-.44-1.648 0l-.979.978c-4.98 4.98-13.084 4.98-18.063 0L1.99.34a1.17 1.17 0 0 0-1.649 0 1.168 1.168 0 0 0 0 1.649l.978.978c4.98 4.98 4.98 13.083 0 18.063l-.977.978c-.221.22-.342.513-.342.825 0 .31.121.604.341.824.442.443 1.21.441 1.65 0l.977-.977c4.98-4.983 13.083-4.98 18.064 0l.978.977c.22.22.513.342.824.342.312 0 .605-.122.824-.342.22-.22.342-.512.342-.824 0-.313-.122-.605-.342-.825l-.977-.978z"/></svg>
```

I outlined the stroke first before scaling it to 24px. Please merge this. I will explain the issue in a separate comment.
![nextra](https://github.com/simple-icons/simple-icons/assets/35000807/b6674c8d-f2e9-43b2-99f5-f7bc54c0f657)